src/Ledger/RolloverLedger/RolloverLedgerContainer.js

```diff
@@ -112,36 +112,42 @@ export const RolloverLedgerContainer = ({ resources, mutator, match, history, lo
   );
 
   const checkIsRolloverAlreadyExist = useCallback(async (currentRollover) => {
-    const { fromFiscalYearId: fromFY, toFiscalYearId: toFY } = currentRollover;
-    const query = `ledgerId==${ledgerId} and rolloverType=="${LEDGER_ROLLOVER_TYPES.commit}" and fromFiscalYearId==${fromFY} and toFiscalYearId==${toFY}`;
-    const { totalRecords } = await ky.get(LEDGER_ROLLOVER_API, { searchParams: { query } }).json();
-
-    if (totalRecords) {
-      const { name: ledgerName } = ledger || {};
-      const fromFYcode = fiscalYears?.find(({ id }) => id === fromFY)?.code;
-      const toFYcode = fiscalYears?.find(({ id }) => id === toFY)?.code;
-
-      const message = intl.formatMessage(
-        {
-          id: 'ui-finance.ledger.rollover.error.conflict',
-        },
-        {
-          ledgerName,
-          fromFYcode,
-          toFYcode,
-        },
-      );
-
-      showCallout({
-        message,
-        type: 'error',
-      });
-
-      return true;
+    try {
+      const { fromFiscalYearId: fromFY, toFiscalYearId: toFY } = currentRollover;
+      const query = `ledgerId==${ledgerId} and rolloverType=="${LEDGER_ROLLOVER_TYPES.commit}" and fromFiscalYearId==${fromFY} and toFiscalYearId==${toFY}`;
+      const { totalRecords } = await ky.get(LEDGER_ROLLOVER_API, { searchParams: { query } }).json();
+
+      if (totalRecords) {
+        const { name: ledgerName } = ledger || {};
+        const fromFYcode = fiscalYears?.find(({ id }) => id === fromFY)?.code;
+        const toFYcode = fiscalYears?.find(({ id }) => id === toFY)?.code;
+
+        const message = intl.formatMessage(
+          {
+            id: 'ui-finance.ledger.rollover.error.conflict',
+          },
+          {
+            ledgerName,
+            fromFYcode,
+            toFYcode,
+          },
+        );
+
+        showCallout({
+          message,
+          type: 'error',
+        });
+
+        return true;
+      }
+
+      return false;
+    } catch (error) {
+      handleRolloverErrors(error);
+
+      return false;
```

Comment:
does it mean we allow rollover when we couldn't check?

Reply:
Good catch. It must be true.